bitnami/argo-cd/templates/server/applications.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.server.additionalApplications }}
```

Comment:
As we allow extraDeploy, not sure if there's much value in this yaml

Reply:
I don't think it hurts to have this file for consistency with the upstream one. You will have both options, using extraDeoply or setting them here.

Reply:
Not sure I like that. If we have a standard way of doing this in all our Bitnami charts, I'm not a fan of giving two options for doing the same operation. I don't think this is a killer feature that we need to copy from the upstream chart and it makes us maintain two ways of providing a same feature. I vote for removing it.